### PR TITLE
Vertical stacked bar chart combined callout and other tweaks

### PR DESCRIPTION
**Pull request by mbest**
#### Pull request checklist

- [x] Include a change request file using `$ yarn change`

#### Description of changes

VerticalStackedBarChart

- props for custom rendering the callout and choosing between combined and per-point callouts
- props for customizing the style or position of the callout
- skip outputting bars for zero-value points
- remove ref array where indexes were not deterministic

## Screenshots

Customize position and display (formatting value)

image

Custom callout for whole stack

image

Design

image

### CODE REVIEW

**Comment by mbest** [comment]
The screenshot above showing a custom combined callout uses this code:

```jsx
          onRenderCalloutPerStack={props => (
            <pre>{JSON.stringify(props, null, 2)}</pre>
          )}
```

Of course, in real use, it would output a pretty version of the data.

**Comment by Raghurk** [comment]
Can you please resolve the conflicts

**Review by Raghurk — APPROVED** [review]

File: packages/charting/src/types/IDataPoint.ts [hunk 1]
```diff
@@ -267,9 +267,10 @@ export interface IVerticalStackedChartProps {
   xAxisPoint: number | string;
 
   /**
-   * chart title for the chart
+   * Callout data for x axis
+   * This is an optional prop, If haven't given, legend will take
    */
-  chartTitle?: string;
+  xAxisCalloutData?: string;
```
Comment (ecraig12345) [inline packages/charting/src/types/IDataPoint.ts:273]: Was this intentional? It seems like `xAxisCalloutData` should have been a *new* prop, not a replacement for `chartTitle`.

File: change/@uifabric-charting-2020-09-04-14-12-33-VerticalStackedBarChart-callout-improvements.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "major",
```
Comment (ecraig12345) [inline change/@uifabric-charting-2020-09-04-14-12-33-VerticalStackedBarChart-callout-improvements.json:2]: I'm pretty sure this should NOT have been a major change. Major version bumps should be reserved for backwards-incompatible API changes, which I don't think I'm seeing in this PR.

File: packages/charting/src/types/IDataPoint.ts (hunk 1 above)
Comment (mbest) [reply packages/charting/src/types/IDataPoint.ts:273]: `chartTitle` was not being used. It's not technically a replacement.

File: change/@uifabric-charting-2020-09-04-14-12-33-VerticalStackedBarChart-callout-improvements.json (hunk 2 above)
Comment (ecraig12345) [reply change/@uifabric-charting-2020-09-04-14-12-33-VerticalStackedBarChart-callout-improvements.json:2]: Never mind, talked to @Raghurk and he said this was intentional

**Pull request merged**